test/support/log_helpers.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Logger.Backend.Humio.LogHelpers do
+  @doc """
+  The Logger timestamp format is weird.  This helper gets you a valid one.
+  """
+  def timestamp do
+    DateTime.utc_now()
+    |> to_logger_timestamp()
+  end
+
+  def to_logger_timestamp(dateTime) do
```

Comment:
pretty sure Elixir naming conventions specify that everything should use `snake_case` except modules with `CamelCase`
```suggestion
  def to_logger_timestamp(date_time) do
```